packages/react-scripts/config/webpack.config.js

```diff
@@ -254,6 +254,11 @@ module.exports = function (webpackEnv) {
       level: 'none',
     },
     optimization: {
+      splitChunks: isEnvDevelopment
+        ? undefined
+        : {
+            chunks: 'all',
+          },
```

Comment:
not sure if we need `name: false` ?

Reply:
It's false by default https://webpack.js.org/plugins/split-chunks-plugin/#splitchunksname

Reply:
Thanks @GraxMonzo I'll resolve then :)